base/permuteddimsarray.jl

```diff
@@ -118,13 +118,82 @@ end
 
 Permute the dimensions of the matrix `m`, by flipping the elements across the diagonal of
 the matrix. Differs from [`transpose`](@ref) in that the operation is not recursive.
+
+# Examples
+```jldoctest
+julia> a = [1 2; 3 4]
+2×2 Array{Int64,2}:
+ 1  2
+ 3  4
+
+julia> b = [5 6; 7 8]
+2×2 Array{Int64,2}:
+ 5  6
+ 7  8
+
+julia> c = [9 10; 11 12]
+2×2 Array{Int64,2}:
+  9  10
+ 11  12
+
+julia> d = [13 14; 15 16]
```

Comment:
Perhaps suppress the output of the `a`, `b`, `c`, and `d` definitions for compactness?